provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -744,6 +757,12 @@ rules:
   - watch
   - get
   - update
+- apiGroups:
+  - "aci.aw"
+  resources:
+  - podifs
+  verbs:
+  - '*'
```

Comment:
Same comment here. There's set of podifs permissions for 'use_aci_anywhere_crd' that should should be expanded to all cases

Reply:
done

Reply:
in the aci_anywhere check, the permissions are '*'. Should that be expanded?